torch/csrc/jit/runtime/static/generated_ops.cpp

```diff
@@ -2637,6 +2637,27 @@ REGISTER_OPERATOR_FUNCTOR(
       return nullptr;
     });
 
+REGISTER_OPERATOR_FUNCTOR(
+    aten::special_log_ndtr,
```

Comment:
not sure why we need this

Reply:
not needed. should be handled by codegen.